lib/vdsm/storage/constants.py

```diff
@@ -242,6 +243,7 @@ def name2type(name):
 # TYPE=PREALLOCATED                           # PREALLOCATED|UNKNOWN|SPARSE
 # VOLTYPE=INTERNAL                            # INTERNAL|SHARED|LEAF
 # GEN=999                                     # int
+# SEQ=4294967295                              # int
 # EOF
 #
 # For more info why this is the worst possible case, see
```

Comment:
Don't we need an update for the text bellow?

```
# On V4 This content requires up to 276 bytes, leaving 236 bytes for the
# description.
#
# On V5 this content requires 270 bytes, leaving 242 bytes for the description
# field.
```

The new key with maximum value needs:

>>> len("SEQ=4294967295\n")
15

So on v5 content can be 285 bytes now, and the description has 227 bytes.

The metadata test should reveal the right value (run it with -s to see the prints).

Reply:
updated:
```
storage/volume_metadata_test.py::TestMDSize::test_max_size[md_params0-4] version=4 type=PREALLOCATED length=486 fields=276 free=26
PASSED
storage/volume_metadata_test.py::TestMDSize::test_max_size[md_params0-5] version=5 type=PREALLOCATED length=495 fields=285 free=17
PASSED
storage/volume_metadata_test.py::TestMDSize::test_max_size[md_params1-4] version=4 type=SPARSE length=484 fields=274 free=28
PASSED
storage/volume_metadata_test.py::TestMDSize::test_max_size[md_params1-5] version=5 type=SPARSE length=492 fields=282 free=20
PASSED
```